### PR TITLE
Events Page

### LINKED ISSUE #19: Events
Screenshot 2024-10-04 at 8 59 27 AM

create the events cards and create mock data for now

### PR DESCRIPTION
**Pull request by Howardwheeler**
Initial Commit of Events Page 
image

### CODE REVIEW

File: src/components/calendar/EventCard.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      <div className="flex justify-center bg-sw-gold-100 bg-gradient-to-r from-sw-gold-100 via-sw-white to-sw-gold-200">
+        <span className="my-[5%] text-3xl font-semibold text-white">
+          {name}
+        </span>
+      </div>
+      <div className="flex flex-col justify-center text-center">
+        <span className="text-3xl font-semibold text-white">{date}</span>
+        <span className="text-3xl font-semibold text-white">{time}</span>
+        <span className="text-3xl font-semibold text-white">{location}</span>
+        <Image src={logo} alt="Logo" className="absolute pt-20" />
```
Comment (stormyy00) [inline src/components/calendar/EventCard.tsx]: try to move the logo on top of the border line with -bottom-2 or [5%] then add a black bg and increase the z-index so it is on top of the border so we can replicate whats on the figma. 

if this doesn't work then leave as is

**Comment by Howardwheeler** [comment]
image
Final look

File: src/components/calendar/EventCard.tsx (hunk 1 above)
Comment (Howardwheeler) [reply src/components/calendar/EventCard.tsx]: The bottom class keeps moving the images to the very bottom so I used margins instead. let me know if there's better alternatives

File: src/components/calendar/EventCard.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  logo: string;
+}) => {
+  return (
+    <div className="flex w-full flex-col border-2 border-white">
+      <div className="flex justify-center bg-sw-gold-100 bg-gradient-to-r from-sw-gold-100 via-sw-white to-sw-gold-200">
+        <span className="my-[5%] text-3xl font-semibold text-white">
+          {name}
+        </span>
+      </div>
+      <div className="flex flex-col justify-center text-center text-3xl font-semibold text-white">
```
Comment (stormyy00) [inline src/components/calendar/EventCard.tsx]: add relative  on the div. so now when you do bottom-0 it will be within that event card

**Comment by Howardwheeler** [comment]
image
look

**Review by stormyy00 — APPROVED** [review]

**Pull request merged**